### PR TITLE
feat: Add Profile Creation

### PR DESCRIPTION
**Pull request by ramos-ph**
Access `POST: /profiles` to create a new profile.

## To-do list

- Add tests.
- Add password encryption.
- Add duplicated account checks.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 
-app.use(express.json())
+const app = express();
 
-app.get('/', (req, res) => res.send({ message: 'ok' }))
+app.use(express.json());
+app.use(routes);
 
-module.exports = app
+app.get('/', (req, res) => res.send({ message: 'ok' }));
```
Comment (noccss) [inline src/app.js]: I think it's valid, send this app.get to routes.

Comment (ramos-ph) [reply src/app.js]: Sure is. Will do it tonight.

Comment (noccss) [reply src/app.js]: Nice, GJ!

**Review by noccss — APPROVED** [review]

File: src/services/profile.service.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const database = require('../database');
+
+exports.create = async (profile) => {
+  await database('profiles').insert(profile);
+
+  const [newProfile] = await database('profiles')
+    .select('*')
+    .where({ username: profile.username })
+    .limit(1);
```
Comment (ramos-ph) [inline src/services/profile.service.js]: Maybe we shouldn't select this again. I think `return profile` would be enough.

**Review by JuanTrindade — APPROVED** [review]
LGTM

**Pull request merged**